tools/dockerfile/interoptest/grpc_interop_ruby/Dockerfile

```diff
@@ -70,12 +70,14 @@ RUN apt-get update && apt-get install -y time && apt-get clean
 RUN gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
 RUN \curl -sSL https://get.rvm.io | bash -s stable
 
-# Install Ruby 2.1
-RUN /bin/bash -l -c "rvm install ruby-2.1"
-RUN /bin/bash -l -c "rvm use --default ruby-2.1"
+# Install Ruby 2.1.0
+# Workaround for https://github.com/grpc/grpc/issues/8905 to let the
+# interop tests run.
+RUN /bin/bash -l -c "rvm install ruby-2.1.0"
```

Comment:
What's difference btw 2.1 and 2.1.0?

Reply:
chatted offline, but FTR this should keep ruby-2.1.0 installed